.github/workflows/publish.yml

```diff
@@ -2,10 +2,8 @@ name: Publish Package
 
 on:
   push:
-    branches:
-      - main
-    paths:
-      - 'packages/shortest/**'
+    tags:
+      - 'v*'
```

Comment:
This will now look for the newly created tag like `v0.2.1` and once the tag is pushed to main like `git push --follow-tag` it will trigger the workflow to publish the package to npm registery.